auth/checkToken.js

```diff
@@ -0,0 +1,19 @@
+const token = require( 'jsonwebtoken' )
+require( 'dotenv' ).config()
+
+module.exports = ( request, response, next ) => {
+  const authToken = request.body.token || request.query.token || request.headers.authorization || request.headers( 'x-access-token' )
```

Comment:
there's a lot going on here - this looks like a separate responsibility (please ensure the lines are less than 80 characters, this currently scrolls off screen for me):
```js
const authToken = request => 
  request.body.token || request.query.token || request.headers.authorization || request.headers( 'x-access-token' )
```

You could also explore making this a promise chain to make it much more readable, and to split up the responsibilities of response into smaller, more concise functions...